js/ui.js

```diff
@@ -0,0 +1,27 @@
+function createCell(content, className) {
+    const cell = document.createElement('td');
+    cell.textContent = content;
+    if (className) {
+        cell.classList.add(className);
+    }
+    return cell;
+}
+
+function createButton(text, className, onClick) {
```

Comment:
Suppose we have 10 parameter, let change this to object
`function createButton({ text, className, onClick }) {}`
By this we can add any number of paramters